### PR TITLE
[hi] improved the grammar in the hindi docs

### PR DESCRIPTION
**Pull request by Vikash-8090-Yadav**
Improved some grammatical mistakes.

### CODE REVIEW

File: README-hi.md [hunk 1]
```diff
@@ -3,11 +3,11 @@
 [![Build Status](https://api.travis-ci.org/kubernetes/website.svg?branch=master)](https://travis-ci.org/kubernetes/website)
 [![GitHub release](https://img.shields.io/github/release/kubernetes/website.svg)](https://github.com/kubernetes/website/releases/latest)
 
-स्वागत हे! इस रिपॉजिटरी में [कुबरनेट्स वेबसाइट और दस्तावेज](https://kubernetes.io/) बनाने के लिए आवश्यक सभी संपत्तियां हैं। हम बहुत खुश हैं कि आप योगदान करना चाहते हैं!
+स्वागत हे! इस रिपॉजिटरी में [कुबरनेट्स वेबसाइट और दस्तावेज](https://kubernetes.io/) बनाने के लिए आवश्यक सभी संपत्तियाँ हैं। हम बहुत खुश हैं कि आप योगदान करने की इच्छा रखते हैं!
```
Comment (kundan2707) [inline README-hi.md:6]: स्वागत है

**Comment by Vikash-8090-Yadav** [comment]
Done with  the chnages. Thanks @kundan2707

**Comment by Vikash-8090-Yadav** [comment]
Any updates on this pr?

**Review by kundan2707 — COMMENTED** [review]
Changes looks good now

**Comment by kundan2707** [comment]
@Vikash-8090-Yadav squash multiple commits

File: README-hi.md [hunk 2]
```diff
@@ -3,11 +3,11 @@
 [![Build Status](https://api.travis-ci.org/kubernetes/website.svg?branch=master)](https://travis-ci.org/kubernetes/website)
 [![GitHub release](https://img.shields.io/github/release/kubernetes/website.svg)](https://github.com/kubernetes/website/releases/latest)
 
-स्वागत हे! इस रिपॉजिटरी में [कुबरनेट्स वेबसाइट और दस्तावेज](https://kubernetes.io/) बनाने के लिए आवश्यक सभी संपत्तियां हैं। हम बहुत खुश हैं कि आप योगदान करना चाहते हैं!
+स्वागत है! इस रिपॉजिटरी में [कुबरनेट्स वेबसाइट और दस्तावेज](https://kubernetes.io/) बनाने के लिए आवश्यक सभी संपत्तियाँ हैं। हम बहुत खुश हैं कि आप योगदान करने की इच्छा रखते हैं!
```
Comment (divya-mohan0209) [inline README-hi.md:6]: More idiomatic in nature. I'd suggest not changing it.
```suggestion
स्वागत है! इस रिपॉजिटरी में [कुबरनेट्स वेबसाइट और दस्तावेज](https://kubernetes.io/) बनाने के लिए आवश्यक सभी संपत्तियाँ हैं। हम बहुत खुश हैं कि आप योगदान करना चाहते हैं!
```

**Review by divya-mohan0209 — CHANGES_REQUESTED** [review]
Thank you for the PR. We should try to keep the sentences as idiomatic as possible rather than complicating it.

**Comment by Vikash-8090-Yadav** [comment]
Sorry, 4 the late response.!
 @divya-mohan0209 please check once I updated all the things as mentioned above. Thanks!

**Review by aj11anuj — APPROVED** [review]
All the requested changes are made properly, and everything seems fine here. Its ready to get merged.

**Review by Babapool — COMMENTED** [review]
Thanks!
/lgtm

**Pull request merged**